doc/ext/traffic-server.py

```diff
@@ -67,12 +79,76 @@ class TSConfVar(std.Target):
         'deprecated': rst.directives.flag,
         'overridable': rst.directives.flag,
         'units': rst.directives.unchanged,
+        'legacy': rst.directives.flag,
     }
     required_arguments = 3
     optional_arguments = 1  # default is optional, special case if omitted
     final_argument_whitespace = True
     has_content = True
 
+    def __generate_code(self, cv_name, cv_default, cv_type):
+        def add_object(config, var, value, type=None):
+            def get_value(type, value):
+
+                def have_multipliers(value, mps):
+                    for m in mps:
+                        if value.endswith(m):
+                            return True
+
+                    return False
+
+                if value is None or value == 'nullptr' or value == 'NULL':
+                    return None
+
+                # We want to make the type right when inserting the element into the dict.
+                if type == 'FLOAT':
+                    return float(value)
+                elif type == 'INT':
```

Comment:
Might it be better to do a conversion and see if that works? Or convert to int and back and see if there's a difference?